app/Http/Middleware/Admin.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+
+class Admin
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure(\Illuminate\Http\Request): (\Illuminate\Http\Response|\Illuminate\Http\RedirectResponse)  $next
+     * @return \Illuminate\Http\Response|\Illuminate\Http\RedirectResponse
+     */
+    public function handle(Request $request, Closure $next)
+    {
+        if(!Auth::user()->admin){
```

Comment:
Gaat deze nu niet stuk als je uitgelogd bent?

Reply:
Als dat zo is kan je het veranderen naar !Auth::user()?->admin 

Als ath::user() null is returned hij false (dat doet dat vraagteken)